### ISSUE TITLE
predict resting-state using saved model

### ISSUE THREAD

**Issue by zhangyu2ustc** [SIMEXP/fmri_predict#15]
as one type of transfer learning, we will save the best model from ALL tasks, and predict the state transition in resting-state data

**Comment by zhangyu2ustc** [comment]
First, retrain the GCN model and save the best one:

'''
from . import checkmat as checkmate
best_ckpt_saver = checkmate.BestCheckpointSaver(save_dir=path,num_to_keep=3,maximize=True,saver=self.op_saver)
best_ckpt_saver.handle(accuracy, sess, self.global_step)
'''

Problem: 
it deletes some of the models because of conflicting of existing saver:
'''
self.op_saver.save(sess, path, global_step=self.global_step)
'''

**Comment by zhangyu2ustc** [comment]
the most common states from resting-state prediction are: motor, language and social

**Comment by zhangyu2ustc** [comment]
guide to restore any pre-trained model and use it for prediction, fine-tuning or further training:
https://cv-tricks.com/tensorflow-tutorial/save-restore-tensorflow-models-quick-complete-tutorial/
